### PR TITLE
Frontpage Illustration and youtube links, both still work in progress

### PR DESCRIPTION
**Pull request by tigert**
Pushing these before the weekend, even though this is not finished yet.

I wanted to do animations on the svg "pipes" but I did not yet managed to make that work, I dont have much experience yet with svg animations and there might be some restrictions on svg interactivity in web browsers, so I need to look into that some more.

Also the columns on the youtube links is a bit broken when the page is resized, will need to revisit that also.

### CODE REVIEW

File: src/content/_index.en.md [hunk 1]
```diff
@@ -11,14 +14,39 @@ As Kubernetes gains adoption, teams are finding they must deploy and manage mult
...
+    <h5> Deploying Submariner
+</h5>
+    <div> {{< youtube cInmBXuZsU8 >}}</div>
+  </div>
+  <div>
+    <h5> Service discovery in Submariner</h5>
+    <div> {{< youtube tXsemQPNhyQ >}}</div>
+  </div>
+</div>
+
```
Comment (mangelajo) [inline src/content/_index.en.md:49]: Ahh. I see this is being omited, please add this to config.toml:

--- a/src/config.toml
+++ b/src/config.toml
@@ -6,6 +6,8 @@ title = "cross cluster network connectivity for Kubernetes"
 theme = "hugo-theme-learn"
 metaDataFormat = "yaml"
 defaultContentLanguageInSubdir= true
+[markup.goldmark.renderer]
+  unsafe = true
 
 [params]
   editURL = "https://github.com/submariner-io/website/edit/master/src/content/"

**Comment by mangelajo** [comment]
Cool, I left you one comment :)

File: src/content/_index.en.md (hunk 1 above)
Comment (tigert) [reply src/content/_index.en.md:49]: added, hopefully I did it correctly.

**Comment by tigert** [comment]
As the commit history got messed up, I am closing this and will make a new one.

**Pull request closed without merging**